content/blog/rustyjam-02/index.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "RustyJam 2"
```

Comment:
❓ The Discord server and the itch.io page have the name as Rusty Jam, rather than RustyJam - should we be consistent here (and below)?